project/profiles/capacity4more/modules/c4m/og/c4m_og_group_menu/c4m_og_group_menu.module

```diff
@@ -109,7 +109,7 @@ function c4m_og_group_menu_form_og_menu_edit_item_form_alter(&$form, &$form_stat
 
   // The select element of the wiki pages should not be displayed on a default
   // menu link.
-  if ($form['link_path']['#value'] != 'wiki') {
+  if ($form['link_path']['#type'] == 'value') {
```

Comment:
Don't think so.
For any default link, there should be no option to link it to wiki page path.
For example, link of News item should point to /news and not any other path.